server/server.dev.js

```diff
@@ -0,0 +1,30 @@
+const express = require('express');
+const bodyparser = require('body-parser');
+const cors = require('cors');
```

Comment:
I don't see cors being used in the server here.

Reply:
Sorry, will finish adding that feature later.